packages/desktop/src/utils/ethereum.js

```diff
@@ -32,14 +32,20 @@ export const getUserAccounts = async (provider) => {
 };
 
 export const signAddress = async (provider, address) => {
-  const message = {
-    address,
-    signed_at: new Date().toISOString(),
-  };
+  const signedAt = new Date().toISOString();
+  const nonce = Math.floor(Math.random() * (999999 - 100000) + 100000);
```

Comment:
if you know a more cryptographic way to do this, go ahead! server just makes sure nonce is in the signed message.

Reply:
Cool! I’ll replace with [`crypto.getRandomValues`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues).